src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx

```diff
@@ -292,45 +296,6 @@ export default function DownloadDialog({
     getSpace()
   }, [installPath, gameInstallInfo?.manifest?.disk_size])
 
-  useEffect(() => {
```

Comment:
duplicated `useEffect`. It was removed on a PR that I closed (#2053) and I forgot to pick the fix before releasing 2.5.0 🤦🏽